src/components/attendee/AttendeeFooter.module.scss

```diff
@@ -1,4 +1,4 @@
-@import "./scss/AttendeeMixins.scss";
+@import "scss/AttendeeMixins.scss";
```

Comment:
How about `@use scss/attendee/mixins.scss` ?

I'm guessing there might be `functions.scss`, `spacing.scss` etc.
And this also goes away with not adding camel case :)

NOTE:

`@use mixins.scss` makes module level items accessible with `mixins.$whitespace` since it has automatic namespace. I suggest using this approach.

The less robust alternative is `@use mixins.scss as *` or similar (can check https://sass-lang.com/documentation/at-rules/use)

Reply:
I chose the robust way because I don't want to go through all of the variables and functions. Check it here: https://github.com/sparkletown/sparkle/pull/2823/commits/22cfe933d386ed598c282f06fdd7d62a1558177f